codec-http/src/main/java/io/netty/handler/codec/http/HttpObjectDecoder.java

```diff
@@ -465,6 +465,10 @@ protected boolean isContentAlwaysEmpty(HttpMessage msg) {
             case 204: case 205: case 304:
                 return true;
             }
+        } else { // assume it is HttpRequest, what else ?
+            if (((HttpRequest) msg).method().equals(HttpMethod.HEAD)) {
+                return true;
```

Comment:
+1. any other conditions we are missing? (e.g TRACE https://tools.ietf.org/html/rfc7231#section-4.3.8)